x-pack/plugins/fleet/server/services/epm/elasticsearch/transform/install.ts

```diff
@@ -151,21 +303,40 @@ async function handleTransformInstall({
       throw err;
     }
   }
-  await esClient.transform.startTransform(
-    { transform_id: transform.installationName },
-    { ignore: [409] }
-  );
+
+  // start transform by default if not set in yml file
+  // else, respect the setting
+  if (startTransform === undefined || startTransform === true) {
+    await esClient.transform.startTransform(
+      { transform_id: transform.installationName },
+      { ignore: [409] }
+    );
+  }
 
   return { id: transform.installationName, type: ElasticsearchAssetType.transform };
 }
 
 const getTransformNameForInstallation = (
   installablePackage: InstallablePackage,
-  path: string,
-  suffix: string
+  transformModuleId: string,
+  suffix: string,
+  assetType?: string
 ) => {
+  // @TODO: Should this be prefixed with `logs`?
+  return `logs-${installablePackage.name}.${transformModuleId}-${
```

Comment:
I might be missing the context here, but I can't see why this should be prefixed with `logs`

Reply:
Thanks for clarifying. Removed this comment here [`fe5ab2f` (#140046)](https://github.com/elastic/kibana/pull/140046/commits/fe5ab2fa6ecbaddf1af20a5413a78dcd70d0a2e6)